Cargo.toml

```diff
@@ -68,8 +68,8 @@ env_logger = "0.7"
 base64 = "^0.11"
 clap = "2.33"
 
-[[example]]
-name = "parse_descriptor"
+# [[example]]
```

Comment:
I think it's better to remove these lines rather than comment them out since the corresponding code was removed.